charts/hedera-the-graph/values-overrides/values-mainnet.yaml

```diff
@@ -3,7 +3,7 @@ index-node:
     chains:
       mainnet:
         providers:
-          - features: []
+          - features: ["archive"]
```

Comment:
Can we add a README docs for configurations and features to capture this.
It's okay to link to the graph docs also

Reply:
Done! ✅